src/display/api.js

```diff
@@ -952,15 +952,51 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
 
     /**
      * @param {getTextContentParameters} params - getTextContent parameters.
+     * @return {ReadableStream} ReadableStream to read textContent chunks.
+     */
+    streamTextContent(params) {
+      const TEXT_CONTENT_CHUNK_SIZE = 100;
+      return this.transport.messageHandler.sendWithStream('GetTextContent', {
+        pageIndex: this.pageNumber - 1,
+        normalizeWhitespace: (params.normalizeWhitespace === true),
+        disableCombineTextItems: (params.disableCombineTextItems !== true),
```

Comment:
This isn't correct, it *needs* to be `combineTextItems: ...`, please see https://github.com/mozilla/pdf.js/pull/8488/files#diff-0ecad279ed2252e3eb47a4d96ec1463cL963 below.